src/parser_urdf.cc

```diff
@@ -1355,46 +1369,46 @@ void URDF2SDF::ParseSDFExtension(TiXmlDocument &_urdfXml)
           sdf->gravity = false;
         }
       }
-      else if (childElem->ValueStr() == "dampingFactor")
+      else if (strcmp(childElem->Name(), "dampingFactor") == 0)
       {
         sdf->isDampingFactor = true;
         sdf->dampingFactor = std::stod(GetKeyValueAsString(childElem));
       }
-      else if (childElem->ValueStr() == "maxVel")
+      else if (strcmp(childElem->Name(), "maxVel") == 0)
       {
         sdf->isMaxVel = true;
         sdf->maxVel = std::stod(GetKeyValueAsString(childElem));
       }
-      else if (childElem->ValueStr() == "minDepth")
+      else if (strcmp(childElem->Name(), "minDepth") == 0)
       {
         sdf->isMinDepth = true;
         sdf->minDepth = std::stod(GetKeyValueAsString(childElem));
       }
-      else if (childElem->ValueStr() == "mu1")
+      else if (strcmp(childElem->Name(), "mu1") == 0)
       {
         sdf->isMu1 = true;
         sdf->mu1 = std::stod(GetKeyValueAsString(childElem));
       }
-      else if (childElem->ValueStr() == "mu2")
+      else if (strcmp(childElem->Value(), "mu2") == 0)
```

Comment:
similar to other comment, use `Name()` instead of `Value()` here (and in the rest of this function)?

Reply:
9c9b74a7